docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/ReplaceMissingValues.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.ML.Data;
+using static Microsoft.ML.Transforms.MissingValueReplacingEstimator.ColumnOptions;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    class ReplaceMissingValues
+    {
+        public static void Example()
+        {
+            // Create a new ML context, for ML.NET operations. It can be used for exception tracking and logging, 
+            // as well as the source of randomness.
+            var mlContext = new MLContext();
+
+            var samples = new List<DataPoint>()
+            {
+                new DataPoint(){ Label = 3, Features = new float[3] {1, 1, 0} },
```

Comment:
If `Label` column is not used in the following transform, we can remove it completely.